magento/pwa-studio
# [PWA-984] - Migrate CheckoutPage graphql operations to peregrine

## Description

### Changes
This PR moves all `.gql` files that are in CheckoutPage components to their corresponding talons in `@magento/peregrine`. 

## Related Issue







Closes PWA-984

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






- Checkout page .gql.js files are moved to Peregrine and no longer reside in Venia
- The Checkout page functions as expected
- Signing in on the Checkout page merges carts correctly
#### Test scenario(s) for direct fix/feature




1. Verify that you can checkout successfully as a guest user
2. Verify that you can checkout successfully as an authorized user
3. Verify that you can come to the checkout page as a guest user, "sign in for express checkout", and that your carts are merged correctly and you can check out successfully

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist
